ui/Screen/Project.svelte

```diff
@@ -113,9 +117,47 @@
   };
 
   fetch({ id: projectId });
-  fetchRevisions({ projectId });
-  $: if ($currentRevision)
-    fetchCommits({ projectId, revision: $currentRevision });
+
+  // Workaround for stores: https://github.com/sveltejs/language-tools/issues/493
+  // TODO(sos): should be removed if/when this is fixed.
+  let revisions: source.PeerRevisions[] | undefined,
+    project: Project | undefined,
```

Comment:
Not too excited about this multi-line let. Makes it hader to edit a line and doesn't seem to be used anywhere else.